packages/sui-consents/src/useUserConsents.js

```diff
@@ -0,0 +1,46 @@
+import {useContext, useEffect, useState} from 'react'
+import SUIContext from '@s-ui/react-context'
+
+import {TCF_WINDOW_API, TCF_VERSION} from './config'
+import hasUserConsents from './hasUserConsents'
+
+/**
+ * @param {array} requiredConsents for example [1, 3, 8]
+ * @return {boolean}
+ */
+export default function useUserConsents(requiredConsents) {
+  /**
+   * Consents acceptance state is inited based on the cookies from the
+   * context, so we know the state of consents from the beginning, even
+   * in SSR.
+   */
+  const {cookies} = useContext(SUIContext)
```

Comment:
I'm not sure `cookies` is available from `SUIContext` on all verticals. 

At least, we should mention this particularity on the README to make this work.

Reply:
Done 👌🏻